### DISCUSSION TITLE
Incorrect "buildings" from Microsoft

### DISCUSSION THREAD

**Discussion by tjukanovt** [OvertureMaps/data#119] [Discussion]
Greetings! 

I have been looking into the Overture data and namely the building layer. One problematic thing I see there is some buildings coming in from Microsoft, which are not actually buildings. These seem to be a mix of containers, trucks, vans and other things which have been classified as buildings. For example in this area in Helsinki, almost all of the small buildings are coming from Microsoft source and are not actually buildings:
https://msbarry.github.io/planetiler-overture-demo/#16.24/60.179561/24.973845/0/25

![image](https://github.com/OvertureMaps/data/assets/13899537/b4f28b7e-435f-4c4e-ba3b-33b34f79f615)

Right now there are no attributes (e.g. ML detection reliability) I could use to filter these out. The issue is very wide spread and I have seen these in multiple geographies. Especially placed on roads, these are very distracting for a map user. 

Would you have any thoughts on how this issue could be addressed? 

I have also opened a ticket upstream, but haven't heard any feedback there: https://github.com/microsoft/GlobalMLBuildingFootprints/issues/87

**Comment by jwass** [comment]
Thanks, @tjukanovt. And thanks for also posting over on the upstream repo.

We've had some discussions around how to collect and process these kinds of issues. @cbeddow This is the kind of thing you're thinking about being able to handle?

**Reply by tjukanovt** [reply]
Thanks for the response @jwass. I think if this kind of artefacts could be eliminated from the data, that would be a real game changer for Overture in terms of added value. My first thought would've been to do some kind of intersection with buffered roads to eliminate the ones which are trucks/containers, but it seems like there are bunch of these also much further away from roads so that might not be an optimal solution.

**Reply by jwass** [reply]
Thanks. We actually do some of that now. ML buildings that overlap water bodies or roads from OSM are not included in the Overture distribution. We noticed that lots of stationary boats are included as buildings in the ML datasets, so they are removed since they're sitting in a water polygon (also included in Overture). But as you're pointing out, it doesn't entirely fix the issue for standalone false buildings. For those - we'd have to collect this in a separate database since there's no way to indicate it in OSM or elsewhere. And that collection system itself could be subject to bad actors that might want to incorrectly flag certain things. We're actively thinking about this.

**Comment by stefankinell** [comment]
I to have the same experience having worked with the Microsoft building data in Sweden before, and now seeing it is part of the Overture data as well. Here there are so many false positives that you just can not use it with confidence. I also see containers, cars etc - that is understandable. But I also get buildings on bare cliffs by the ocean, randomly in the forest, in the mountains in the north of Sweden. They are just not reliable. 

I have also reported it directly to the Microsoft building projects but it seems that our regions is not very interesting to work with since there is never a reply. But I think it is important for those working with the overture project to know that Microsoft building data is not reliable.

**Reply by andwoi** [reply]
This is good feedback. We've certainly seen false positives in rural areas before, but it sounds like you're encountering more than expected in Sweden. If you think of it, create [a new issue](https://github.com/microsoft/GlobalMLBuildingFootprints/issues/new) with details like location and we'll use the feedback to improve.

**Reply by stefankinell** [reply]
Thanks for the link. I posted some examples and a file of 6913 false positives in my home area as an issue at https://github.com/microsoft/GlobalMLBuildingFootprints/issues/94
